datastore/transaction.go

```diff
@@ -110,17 +110,25 @@ func (c *Client) NewTransaction(ctx context.Context, opts ...TransactionOption)
 	return c.newTransaction(ctx, newTransactionSettings(opts))
 }
 
-func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (*Transaction, error) {
+func (c *Client) newTransaction(ctx context.Context, s *transactionSettings) (_ *Transaction, err error) {
```

Comment:
Why is the return signature changed? Seems unnecessary.

Reply:
I used `err` in return signature for this statement `defer func() { trace.EndSpan(ctx, err) }()` like it was done in this [CL](https://code-review.googlesource.com/c/gocloud/+/39590/1..5).
I could revert it too, if it's not okay

Reply:
Ahh okay. This looks fine, then.